### PR TITLE
fix for Nutch-2011 contributed b Sujen Shah

### PR DESCRIPTION
**Pull request by sujen1412**

### CODE REVIEW

File: src/java/org/apache/nutch/fetcher/FetchNode.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package org.apache.nutch.fetcher;
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/fetcher/FetchNode.java]: Can you add ALv2 headers?

File: src/java/org/apache/nutch/fetcher/FetchNode.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+  public long getFetchTime() {
+    return fetchTime;
+  }
+  public void setFetchTime(long fetchTime) {
+    System.out.println(this.hashCode() + " Setting fetchTime to : " + fetchTime);
+    this.fetchTime = fetchTime;
+  }
+  
+//  public String toString(){
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/fetcher/FetchNode.java]: If it's commented out, don't include it please :)

File: src/java/org/apache/nutch/fetcher/FetchNodeDb.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+public class FetchNodeDb {
+
+  private Map<Integer, FetchNode> fetchNodeDbMap;
+  private int index;
+  private static FetchNodeDb fetchNodeDbInstance = null;
+  
+  public FetchNodeDb(){    
+//    System.out.println("Calling FetchNode constructor");
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/fetcher/FetchNodeDb.java]: remove

File: src/java/org/apache/nutch/fetcher/FetchNodeDb.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    fetchNodeDbMap = new ConcurrentHashMap<Integer, FetchNode>();
+    index = 1;
+  }
+  
+  public static FetchNodeDb getInstance(){
+    
+    if(fetchNodeDbInstance == null){
+//      System.out.println("Creating FetchNode instance");
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/fetcher/FetchNodeDb.java]: remove

File: src/java/org/apache/nutch/fetcher/FetchNodeDb.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      fetchNodeDbInstance = new FetchNodeDb();
+    }
+//    System.out.println("FetchNodeDb Instance : " + fetchNodeDbInstance);
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/fetcher/FetchNodeDb.java]: remove

File: src/java/org/apache/nutch/service/model/response/FetchNodeDbInfo.java [hunk 6]
```diff
@@ -0,0 +1,87 @@
+package org.apache.nutch.service.model.response;
```
Comment (chrismattmann) [inline src/java/org/apache/nutch/service/model/response/FetchNodeDbInfo.java]: ALv2 header

**Comment by sujen1412** [comment]
Updated the files

**Comment by chrismattmann** [comment]
thanks @sujen1412 will look to commit

**Pull request closed without merging**